magstock/_version.py

```diff
@@ -0,0 +1,2 @@
+from __future__ import unicode_literals
+__version__ = '5.0'
```

Comment:
This isn't really required, but it's generally considered good practice for Python modules to have a version, and Sideboard modules display that version in some circumstances.